curvefs/conf/client.conf

```diff
@@ -109,39 +109,38 @@ volume.bigFileSize=1048576
 volume.volBlockSize=4096
 volume.fsBlockSize=4096
 
-# alloator type, supported {bitmap}
+# allocator type, supported {bitmap}
 volume.allocator.type=bitmap
 
 ## for bitmap allocator
 # size of each bit, default is 4MiB
-volume.bitmapallocator.size_per_bit=4194304
+volume.bitmapAllocator.size_per_bit=4194304
 
 # small allocation proportion [0-1]
-volume.bitmapallocator.small_alloc_proportion=0.2
+volume.bitmapAllocator.small_alloc_proportion=0.2
```

Comment:
please check for changes to the configuration item name to ensure that every where that uses the configuration item has changed.

ditto to other item

Reply:
updated & checked other item : )